static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,333 @@
         }
     }
 }
+
+// New gallery stuff
+
+// TODO: put in pasteup
+$multimedia-support-5: #161616;
+
+// The main image needs to take up 100vh and the slim header needs to sit just on top of it.
+.l-header--new-gallery {
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    left: 0;
+}
```

Comment:
Out of curiosity, should the image appear as 100vh inclusive or exclusive of the header? Currently it's inclusive of the header if I understand correctly?

If it's the former, this approach might be problematic on devices with a landscape orientation:

![image](https://cloud.githubusercontent.com/assets/921609/14670459/2a460b92-06e5-11e6-8f92-5e07d8b83168.png)